<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDialog</class>
 <widget class="QDialog" name="LoginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QPushButton" name="pushButtonLogin">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>220</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>登录</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>70</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>用户名：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>120</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>密码：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPassWord">
   <property name="geometry">
    <rect>
     <x>127</x>
     <y>120</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxRemeber">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>169</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>记住密码</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxAuto">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>167</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>自动登录</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxUserName">
   <property name="geometry">
    <rect>
     <x>127</x>
     <y>70</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonInsertData">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>70</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>注册帐号</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonBackData">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>120</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>找回密码</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelError">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>200</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
